### PR TITLE
CFD-294 : Homepage : My Groups (Logged-in users) - theming.

### PR DESCRIPTION
**Pull request by nader77**
Follow up on #503

### CODE REVIEW

**Comment by nader77** [comment]
This will hide "Show all" button when "My groups" are 3 or less.
selection_064

Also fixes "See more" link in "Suggested groups".

**Comment by nader77** [comment]
@HelenaEksler Please review.

File: capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my-groups.tpl.php [hunk 1]
```diff
@@ -6,10 +6,12 @@
       <div class="collapse" id="allGroups">
         <?php print $extra_groups ?>
       </div>
-      <a class="see-more-link" href="javascript://" data-toggle="collapse" data-target="#allGroups" aria-expanded="false" aria-controls="allGroups">
+      <?php if ($show_all_link) : ?>
+      <a class="see-more-link" id="toggleMyGroups" href="javascript://" data-toggle="collapse" data-target="#allGroups" aria-expanded="false" aria-controls="allGroups">
```
Comment (HelenaEksler) [inline capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my-groups.tpl.php]: indentation
```php
<?php if ($show_all_link) : ?>
  <a class="see-more-link" id="toggleMyGroups" href="javascript://" data-toggle="collapse" data-targ...
```

**Comment by HelenaEksler** [comment]
@nader77  Fix this and assign to Peter.

**Comment by nader77** [comment]
@zero2one Ready for code review.

**Pull request merged**